tests/_Str.js

```diff
@@ -0,0 +1,63 @@
+import { assert } from 'chai';
+import * as _Str from '../src/fe/implicit/_Str';
+
+const {
+  isTrue,
+  isFalse,
+  deepEqual: deep,
+  notDeepEqual: notDeep,
+  equal,
+} = assert;
+
+describe('_Str', () => {
+  describe('pad', () => {
+    it('Check padding on both sides', () => {
+      const arr = _Str.pad('test').split('');
+      const hasPadding = arr[0] == ' ' && arr[arr.length - 1] == ' ';
+      isTrue(hasPadding);
+    });
+  });
+
+  describe('contains', () => {
+    it('Check if a string contains a substring', () => {
+      const str = 'hello world';
+      const substr = 'world';
+      const contains = _Str.contains(str, substr);
+      isTrue(contains);
+    });
+
+    it('Check if a string does not contain a substring', () => {
+      const str = 'hello world';
+      const substr = 'world1';
+      const contains = _Str.contains(str, substr);
+      isFalse(contains);
+    });
+  });
+
+  describe('slice', () => {
```

Comment:
Add tests where `from` is negative and another where `to` is greater than the length